### PR TITLE
Fix shop protection and upgrades protection not work

### PR DESCRIPTION
**Pull request by ImYenil**
Fixed bug where shop protection and upgrades protection did not work

### CODE REVIEW

**Comment by ImYenil** [comment]
Fixed

File: bedwars-api/src/main/java/com/andrei1058/bedwars/api/configuration/ConfigPath.java [hunk 1]
```diff
@@ -163,6 +163,7 @@ public class ConfigPath {
 
     public static final String ARENA_DISPLAY_NAME = "display-name";
     public static final String ARENA_CONFIGURATION_MAX_BUILD_Y = "max-build-y";
+    public static final String ARENA_CONFIGURATION_MIN_BUILD_Y = "min-build-y";
```
Comment (tomkeuper) [inline bedwars-api/src/main/java/com/andrei1058/bedwars/api/configuration/ConfigPath.java:166]: Whats this used for?

Comment (ImYenil) [reply bedwars-api/src/main/java/com/andrei1058/bedwars/api/configuration/ConfigPath.java:166]: > Whats this used for?

just mistake

Comment (tomkeuper) [reply bedwars-api/src/main/java/com/andrei1058/bedwars/api/configuration/ConfigPath.java:166]: used in #87

**Pull request merged**